x/slashing/client/rest/query.go

```diff
@@ -64,7 +59,7 @@ func signingInfoHandlerListFn(cliCtx context.CLIContext, storeName string, cdc *
 
 		_, page, limit, err := rest.ParseHTTPArgs(r)
 		if err != nil {
-			rest.WriteErrorResponse(w, http.StatusBadRequest, err.Error())
```

Comment:
this is also correct and shouldn't be removed, since this checks for badly parsed query parameters

Reply:
Noted. Reverted